web/sendgrid_webhooks.py

```
@@ -0,0 +1,188 @@
+        profile.save(
+            update_fields=[
+                "email_delivered_count",
+                "email_open_count",
+                "email_click_count",
+                "email_bounce_count",
+                "email_drop_count",
+                "email_spam_report_count",
+                "last_email_event",
+                "last_email_event_time",
+                "email_last_event_data",
+            ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)